nbdime/prettyprint.py

```diff
@@ -112,6 +112,8 @@ def should_ignore_path(self, path):
             return not self.attachments
         if starred.startswith('/cells/*/metadata') or starred.startswith('/metadata'):
             return not self.metadata
+        if starred.startswith('/id'):
+            return not self.id
```

Comment:
I'm not sure if this makes sense. The path would need to be `/cells/*/id` for it to match cell ids.

Reply:
Indeed, this was wrong. I changed it to` /cells/*/id`.